src/rmm.cpp

```diff
@@ -85,6 +82,15 @@ rmmError_t rmmFinalize()
     
     return RMM_SUCCESS;
 }
+
+// Query the initialization state of RMM.
+bool rmmIsInitialized(rmmOptions_t *options)
+{
+    if (options) {
```

Comment:
You used this style in `initialize`, and I find it far more readable.

```suggestion
    if (nullptr != options) {
```